### PR TITLE
add RemoveScroll component to ApplicationNavigationBar

### PR DESCRIPTION
**Pull request by Lxkas**
This pull request adds the RemoveScroll component to the ApplicationNavigationBar, which ensures that the navigation bar remains scrollable even when the user is scrolling the page. This improves the user experience and makes the navigation bar more accessible.

### CODE REVIEW

File: src/components/core/ApplicationNavigationBar.tsx [hunk 1]
```diff
@@ -10,18 +10,20 @@ interface ApplicationNavigationBarProps {
 export default function ApplicationNavigationBar({ opened }: ApplicationNavigationBarProps) {
 	return (
 		<AppShell.Navbar p="md" hidden={!opened}>
-			<Stack className="h-full" gap="xs">
-				{navigationItems.map(({ title, href, subitems }) => (
-					<RecursiveNavLink key={title} item={{ title, href, subitems }} />
-				))}
+			<RemoveScroll enabled={opened}>
```
Comment (chinathaip) [inline src/components/core/ApplicationNavigationBar.tsx:13]: I couldn't find any documents from mantine, but essentially everything wrapped inside this component will be scrollable, the rest would not?

https://github.com/theKashey/react-remove-scroll

**Comment by chinathaip** [comment]
Pretty cool,

we're actually scrolling the page behind navbar in the current version:

https://github.com/stamford-syntax-club/stamfordcenter-frontend/assets/92321280/d1080fc1-072d-40cd-aa2d-f50792da24ea

whereas the new one prevents it from happening:

https://github.com/stamford-syntax-club/stamfordcenter-frontend/assets/92321280/dd9cafd9-72e1-493b-8d0f-352d4d6141c2

File: src/components/core/ApplicationNavigationBar.tsx (hunk 1 above)
Comment (Lxkas) [reply src/components/core/ApplicationNavigationBar.tsx:13]: Yeah

**Comment by Lxkas** [comment]
> Pretty cool,
> 
> we're actually scrolling the page behind navbar in the current version:
> 
>  RPReplay_Final1699114088.MP4 
> whereas the new one prevents it from happening:
> 
>  RPReplay_Final1699114108.MP4

That's the point XD

**Review by chinathaip — APPROVED** [review]

**Pull request merged**